profiles/coreos/base/package.use

```diff
@@ -39,7 +39,7 @@ net-analyzer/nmap ncat -lua
 app-admin/sudo -sendmail
 
 # avoid pulling in gnutls, disable gentoo-only bits, enable journal upload
-sys-apps/systemd -ssl curl vanilla lzma
+sys-apps/systemd -ssl curl vanilla -lz4 lzma gcrypt
```

Comment:
I was hoping to make few changes to the upstream ebuild. Gentoo adopted lz4, while everyone else went with lzma.